htdocs/langs/en_US/cashdesk.lang

```diff
@@ -153,3 +153,4 @@ LineDiscount=Line discount
 LineDiscountShort=Line disc.
 InvoiceDiscount=Invoice discount
 InvoiceDiscountShort=Invoice disc.
+ConfirmClosePayment=Are you sure you want to delete this payment ?
```

Comment:
The message ask to confirm to delete payment, but in code, the message is ask when we try to enter the payment popup so to create a payment (so the opposite). Did i miss something ?

Reply:
it replaces the colorbox.close function of the payment colorbox, so it it called when you close the payment box.